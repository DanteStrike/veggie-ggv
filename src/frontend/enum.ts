export const enum Position {
  AFTERBEGIN = `afterbegin`,
  BEFOREEND = `beforeend`
}

export const enum RestaurantMenuOffersGroup {
  STARTERS =`starters`,
  MAIN_DISHES =`main-dishes`,
  DESSERTS =`desserts`,
  DRINKS =`drinks`,
}
