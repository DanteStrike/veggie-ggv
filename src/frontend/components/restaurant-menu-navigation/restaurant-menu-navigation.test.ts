import RestaurantMenuNavigation from "./restaurant-menu-navigation";
import {RestaurantMenuOffersGroup} from "../../enum";

it(`Restaurant Menu Navigation component should render correctly`, () => {
  const component = new RestaurantMenuNavigation(RestaurantMenuOffersGroup.STARTERS, () => {});
  expect(component.getElement()).toMatchSnapshot();
});
