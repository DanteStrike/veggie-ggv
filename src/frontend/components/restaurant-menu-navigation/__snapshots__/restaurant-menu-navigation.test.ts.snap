// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Restaurant Menu Navigation component should render correctly 1`] = `
<ul
  class="nav__list nav__list--restaurant-menu"
>
  
        
  <li
    class="nav__item"
  >
    
          
    <button
      class="button restaurant-menu__button nav__link nav__link--current"
      data-group="starters"
    >
      Starters
    </button>
    
        
  </li>
  
        
  <li
    class="nav__item"
  >
    
          
    <button
      class="button restaurant-menu__button nav__link"
      data-group="main-dishes"
    >
      Main Dishes
    </button>
    
        
  </li>
  
        
  <li
    class="nav__item"
  >
    
          
    <button
      class="button restaurant-menu__button nav__link"
      data-group="desserts"
    >
      Desserts
    </button>
    
        
  </li>
  
        
  <li
    class="nav__item"
  >
    
          
    <button
      class="button restaurant-menu__button nav__link"
      data-group="drinks"
    >
      Drinks
    </button>
    
        
  </li>
  
      
</ul>
`;
